tests/lib/src/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set (TARGET_NAME "tests_shared_lib")
+
+file (GLOB SRC *.cpp)
+add_library(${TARGET_NAME} STATIC ${SRC})
+
+target_include_directories(${TARGET_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+
+set(OpenVINO_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../")
+
+# Search OpenVINO Inference Engine via InferenceEngine_DIR
+find_package(IEDevScripts REQUIRED
+             PATHS "${OpenVINO_SOURCE_DIR}/cmake/developer_package"
+             NO_CMAKE_FIND_ROOT_PATH
+             NO_DEFAULT_PATH)
+
+# Search OpenVINO Inference Engine installed
+find_package(InferenceEngine)
```

Comment:
@ishariko let's try to use `find_package(InferenceEngine)` only - it means OpenVINO should be installed via `make install` and available in cmake. It will help with unifying of different ways of Inference Engine inclusion

Note that some functionality (like `IE_THROW`) may be not available, so it would be better to replace with new one

Reply:
@vurusovs done. I used your [PR](https://github.com/openvinotoolkit/openvino/pull/6744) for solve this comment